mkosi/config.py

```diff
@@ -994,6 +995,12 @@ class MkosiConfigParser:
             parse=config_parse_feature,
             help="Use btrfs subvolumes for faster directory operations where possible",
         ),
+         MkosiConfigSetting(
+            dest="seed",
+            section="Output",
+            parse=config_parse_string,
```

Comment:
Can you add config_parse_seed() for this and do a basic check on whether the value is a proper UUID?